test/go-tests/test_utils.go

```diff
@@ -168,32 +169,35 @@ func GetInternalKeptnAPI(ctx context.Context, internalService, localPort string,
 	return keptnInternalAPI, nil
 }
 
-func CreateProject(projectName, shipyardFilePath string, recreateIfAlreadyThere bool) error {
+func CreateProject(projectName string, shipyardFilePath string, recreateIfAlreadyThere bool) (string, error) {
+
 	retries := 5
 	var err error
 	var resp *req.Resp
+	newProjectName := osutils.GetOSEnv(KeptnNamespacePrefix) + "-" + projectName
```

Comment:
I would add something like the following to explain this line of code and add a default to don't have projects starting with `-` when tests are run locally
```suggestion
	// this is to avoid name aliasing with multiple CI parallel run of the integration tests
	newProjectName := osutils.GetOSEnvOrDefault(KeptnNamespacePrefix, DefaultKeptnNamespace) + "-" + projectName
```

Reply:
fixed